Common/include/option_structure.hpp

```diff
@@ -71,6 +71,26 @@ enum class SU2_COMPONENT {
   SU2_SOL  /*!< \brief Running the SU2_SOL software. */
 };
 
+/*!
+ * \namespace cudaKernelParameters
+ * \brief Namespace that contains variables and helper functions that are utilized to calculate CUDA Kernel parameters.
+ * \author A. Raj
+ */
+namespace cudaKernelParameters{
+
+  /*!
+   * \brief Returns the rounded up value of the decimal quotient to the next integer (in all cases).
+   */
+  inline unsigned int rounded_up_division(int divisor, int dividend) { return ((dividend + divisor - 1) / divisor); }
+
+  /*!
+   * \brief Returns the rounded down value of the decimal quotient to the previous integer (in all cases).
+   */
+  inline unsigned int rounded_down_division(int divisor, int dividend) { return ((dividend - divisor + 1) / divisor); }
```

Comment:
There are similar helpers in omp_structure.h, move them somewhere you can re use.